packages/provider-elasticsearch/src/example-types/metricGroups/pivot.js

```diff
@@ -137,10 +146,10 @@ let createPivotScope = (node, schema, getStats) => {
       _.flatMap('loaded')
     )(expansions)
 
-  let isFirstOfType = (expansion) =>
+  let isFirstOfType = expansion =>
```

Comment:
@stellarhoof these seems like prettier diffs - we should have one config for the monorepo and run it for everything in its own PR

Reply:
We do, but nothing's running in CI because we need to merge https://github.com/smartprocure/contexture/pull/2 since I deleted my personal token from this repo.